lib/account.js

```diff
@@ -43,141 +49,171 @@ function parseMessage(message, callback) {
   });
 }
 
-exports.register = function(server, options, next) {
+const register = function(server, options) {
```

Comment:
change to the Hapi register API. See at the bottom of the file for what it has turned into.